### ISSUE TITLE
Color contrast issues

### ISSUE THREAD

**Issue by chriswales95** [LadybirdBrowser/ladybird.org#104]
# Issue

Several areas of the Ladybird browser website have low color contrast.

The Web Content Accessibility Guidelines says that text (and images of text) should have an appropriate contrast ratio  -- defined in WCAG 2.2 section 1.4.3: https://www.w3.org/TR/WCAG22/#contrast-minimum. This helps people with visual impairments who use the website as some may struggle with insufficient contrast.

It's a relatively easy thing to fix. It could be addressed by subtly changing the foreground text colour. For example, the footer text could be changed from to #404040 #757575. This would allow it to pass at the AA level.

# Examples

## News & Announcements

<img width="1399" alt="image" src="https://github.com/user-attachments/assets/5d4fbf95-e335-4f36-847f-733eef44d4f3">

## Footer text

<img width="1399" alt="image" src="https://github.com/user-attachments/assets/2da39934-8fc4-4619-97b0-4293fa9b8333">

**Comment by chriswales95** [comment]
I'm happy to have a look at some of the contrast issues.

**Comment by chriswales95** [comment]
Will close since it was merged. Thanks!
